storage/client.go

```diff
@@ -313,6 +314,13 @@ type restoreObjectParams struct {
 	copySourceACL  bool
 }
 
+type moveObjectParams struct {
+	bucket, srcObject, dstObject string
+	srcConds                     *Conditions
+	dstConds                     *Conditions
+	encryptionKey                []byte
```

Comment:
Do we require predefinedACL, sendCRC32C in moveObject?

Reply:
No -- MoveObject does not change the ACL or need checksumming (b/c object data does not change, only the name)